src/platforms/components/PolygonIdReward.tsx

```diff
@@ -0,0 +1,113 @@
+import { Icon, Spinner, Tooltip, useDisclosure } from "@chakra-ui/react"
+import { useOpenJoinModal } from "components/[guild]/JoinModal/JoinModalProvider"
+import {
+  RewardDisplay,
+  RewardIcon,
+  RewardProps,
+} from "components/[guild]/RoleCard/components/Reward"
+import AvailibiltyTags, {
+  getTimeDiff,
+} from "components/[guild]/RolePlatforms/components/PlatformCard/components/AvailibiltyTags"
+import useAccess from "components/[guild]/hooks/useAccess"
+import useGuild from "components/[guild]/hooks/useGuild"
+import useIsMember from "components/[guild]/hooks/useIsMember"
+import Button from "components/common/Button"
+import { ArrowSquareOut, LockSimple } from "phosphor-react"
+import { MintPolygonIdProofModal } from "platforms/PolygonId/components/MintPolygonIdProofModal"
+import platforms from "platforms/platforms"
+import { useMemo } from "react"
+import { PlatformType } from "types"
+import { useAccount } from "wagmi"
+
+const PolygonIdReward = ({ platform, withMotionImg }: RewardProps) => {
```

Comment:
We can remove capacity/time related stuff here, that's only applicable for text-based rewards & NFT rewards